src/modules/users/users.controller.ts

```diff
@@ -36,18 +38,17 @@ const getUser = async (req, res) => {
 
 const getUsers = async (req, res) => {
   try {
-    const users = await userService.find();
+    const users = await userService.find(req.params);
     res.status(200).send(users);
   } catch (error) {
     handleError(res, error);
   }
 };
 
 const updateUser = async (req, res) => {
-  const { name, age, email, tel, role } = req.body;
   try {
-    const user = await userService.update(req.params.id, { name, age, email, tel, role });
-    res.status(200).json(user);
+    const updatedUser = await userService.update(req.body);
```

Comment:
you have bug here, you have to get id from parameters and all other fields from body

- [ ] fix this